resources/assets/js/helpers/textToParagraphs.tsx

```diff
@@ -9,17 +9,22 @@ import React from "react";
 export const textToParagraphs = (
   text: string,
   props?: any,
+  specificProps?: object,
```

Comment:
I'd like to see an explanation of this arg added to the docblock, and a more accurate typing.